### PR TITLE
CASMINST-6286-release-1.3 move ceph upgrade before storage node upgrade in csm-1.3

### PR DESCRIPTION
**Pull request by leliasen-hpe**
# Description
A problem was hit running upload_ceph_images_to_nexus during CSM 1.2 to CSM 1.3 upgrade. This issue occurred because the script uses ceph commands that are not valid for the version of Ceph running in CSM 1.2 (ceph version v15.2.15). The purpose of this script is to put the ceph images in Nexus and restart Ceph daemons to use the nexus image. Without this, the daemons won't start after reboot because can't find the image they are looking for (specifically, the sha needs to be the same or they fail). 

Because this script is not valid in CSM-1.2, the ceph upgrade was moved from Stage 4 to Stage 1. The ceph daemons will be upgraded to the ceph version in nexus and then the storage nodes will be upgrade. After reboot, the storage daemons will start fine because they are referencing the image in Nexus.

# Checklist

- [x] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [x] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

 
[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

**Review by bklei — APPROVED** [review]

**Review by cdelatte-hpe — APPROVED** [review]

File: upgrade/Stage_1.md [hunk 1]
```diff
@@ -41,37 +41,68 @@ Ceph can begin to exhibit latency over time when upgrading the cluster from prev
...
+Before starting [Storage node image upgrade](#storage-node-image-upgrade), access the Argo UI to view the progress of this stage.
+Note that the progress for the current stage will not show up in Argo before the storage node image upgrade script has been started.
+
+For more information, see [Using the Argo UI](../operations/argo/Using_the_Argo_UI.md) and [Using Argo Workflows](../operations/argo/Using_Argo_Workflows.md).
+
+## CSM Upgrade requirement for upgrades to a new CSM release version
+
+**IMPORTANT:**
+
+> If the upgrade is to a new CSM release, e.g. `CSM-1.2.0` to `CSM-1.3.0`, then you will need to run the following to upgrade Ceph. This will upgrade Ceph from `v15.2.15` to `v16.2.9`.
```
[the remark this answers is not in the document]
Comment (leliasen-hpe) [reply upgrade/Stage_1.md:56]: Fixed.

**Review by rustydb — APPROVED** [review]

File: upgrade/Stage_1.md [hunk 2]
```diff
@@ -41,37 +41,86 @@ Ceph can begin to exhibit latency over time when upgrading the cluster from prev
...
 > The issue stems from slightly different `sha` values for the Ceph containers for in-family CSM storage node images which will prevent the Ceph containers from starting.
-> This will utilize the upgrade procedure to accomplish this as it has built in checks and health monitoring to better manage this rolling restart of the Ceph containers with the image stored in Nexus.  Please see [`cubs_tool` usage for further information](../operations/utility_storage/Cubs_tool_Usage.md)
 
-(`ncn-s001#`) In family upgrade to a new container with the same Ceph version:
+(`ncn-s001#`) Upload Ceph container images into Nexus and restart daemons so that they use the image in Nexus.
 
-```bash
-/usr/share/doc/csm/scripts/cubs_tool.py --version v16.2.9 --registry registry.local/artifactory.algol60.net/csm-docker/stable/quay.io --upgrade --in_family_override
-```
+  ```bash
```
Comment (rustydb) [inline upgrade/Stage_1.md:120]:
```suggestion
   ```bash
```

Comment (rustydb) [inline upgrade/Stage_1.md:120]:
```suggestion
```bash
```

File: upgrade/Stage_1.md [hunk 3]
```diff
@@ -41,37 +41,86 @@ Ceph can begin to exhibit latency over time when upgrading the cluster from prev
...
+   scp ncn-m001:/usr/share/doc/csm/scripts/upload_ceph_images_to_nexus.sh /srv/cray/scripts/common/upload_ceph_images_to_nexus.sh
```
Comment (rustydb) [inline upgrade/Stage_1.md:121]:
```suggestion
scp ncn-m001:/usr/share/doc/csm/scripts/upload_ceph_images_to_nexus.sh /srv/cray/scripts/common/upload_ceph_images_to_nexus.sh
```

File: upgrade/Stage_1.md [hunk 4]
```diff
@@ -41,37 +41,86 @@ Ceph can begin to exhibit latency over time when upgrading the cluster from prev
...
+   /srv/cray/scripts/common/upload_ceph_images_to_nexus.sh
+   ```
```
Comment (rustydb) [inline upgrade/Stage_1.md:123]:
```suggestion
```bash
scp ncn-m001:/usr/share/doc/csm/scripts/upload_ceph_images_to_nexus.sh /srv/cray/scripts/common/upload_ceph_images_to_nexus.sh
/srv/cray/scripts/common/upload_ceph_images_to_nexus.sh
```
```

**Review by rustydb — APPROVED** [review]

**Pull request merged**